composer.json

```
@@ -18,16 +18,16 @@
         "source": "https://github.com/cakephp/queue"
     },
     "require": {
-        "php": ">=7.2.0",
-        "cakephp/cakephp": "^4.1",
+        "php": ">=7.4.0",
+        "cakephp/cakephp": ">=4.1 ",
```

```suggestion
        "cakephp/cakephp": "^4.1 ",
```

Can we revert this?  This isn't a safe range to use, as it will include 5.x which will not be compatible.